lib/utils/getInterestOptions.ts

```diff
@@ -19,7 +19,7 @@ const interests = [
   "hacktoberfest",
   "clojure",
 ] as const;
-export type interestsType = (typeof interests)[number];
+export type InterestType = (typeof interests)[number];
```

Comment:
Renamed this to Pascal case as it's a type. This avoids confusion with variables in the code.